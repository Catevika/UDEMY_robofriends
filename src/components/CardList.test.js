import React from 'react';
import Cardlist from './CardList';
import { shallow } from 'enzyme'; // mount and render can also be used - see Enzyme documentation

describe('CardList component', () => {
	it('expects to render Cardlist component', () => {
		const mockRobots = [
			{
				id: 1,
				name: 'John Snow',
				email: 'john@gmail.com'
			}
		];
		expect(shallow(<Cardlist robots={mockRobots} />).debug()).toMatchSnapshot();
	});
});
