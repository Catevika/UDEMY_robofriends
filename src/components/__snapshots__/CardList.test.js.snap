// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList component expects to render Cardlist component 1`] = `
"<div>
  <Card id={1} name=\\"John Snow\\" email=\\"john@gmail.com\\" />
</div>"
`;
